SU-SWS/stanford_components
# EARTH-519 Changed svg title

# READY FOR REVIEW

# Summary
- Changed the svg title to the same as the link title. It seems the tooltip when hovering over the link uses the svg title instead of the title attribute.

# Needed By (Date)
- end of sprint

# Urgency
- low

# Steps to Test

1. checkout branch
2. clear cache
3. review patters page and hover over the arrow to see its text.

# Associated Issues and/or People
- https://github.com/stanford-earth/matson/pull/82
- https://github.com/stanford-earth/stanford_news/pull/8

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
